src/main/java/io/vertx/circuitbreaker/impl/CircuitBreakerImpl.java

```diff
@@ -126,11 +131,7 @@ public synchronized CircuitBreaker reset() {
   private synchronized void sendUpdateOnEventBus() {
     String address = options.getNotificationAddress();
     if (address != null) {
-      vertx.eventBus().publish(address, new JsonObject()
-          .put("name", name)
-          .put("state", state)
-          .put("failures", failures)
-          .put("node", vertx.isClustered() ? ((VertxInternal) vertx).getClusterManager().getNodeID() : "local"));
+      vertx.eventBus().publish(address, metrics.toJson());
```

Comment:
I think it is correct. Say that you have 2 consumers a real time app over sockjs and another connector then send would only deliver to one address.